### ISSUE TITLE
Entire page paints on scrolling

### ISSUE THREAD

**Issue by ebidel** [Polymer/core-header-panel#27]
In Chrome 41, similar to Polymer/core-scroll-header-panel#38.

**Comment by ericeslinger** [comment]
FWIW (see my related comment), this breaks the usage of position: fixed elements as block children of the core-header-panel (what I want to do is inject that element so it lives below my core-header-panel in the DOM tree for variable binding reasons, but renders on top of a bunch of other stuff).

**Comment by kbwatts** [comment]
Had the same problem because we use a persistent footer and position:fixed which broke (and you can't avoid upgrading to 0.5.6 because of the bowers using ^0.5 which forces you go to the highest 0.5.\* possible).

I'm sure this is not the best way, but our temporary solution was to overwrite these using 

```
html /deep/ #mainContainer {
  -webkit-transform: none !important;
  transform: none !important;
}
```

**Comment by danieleliberti** [comment]
The fixes [ 9f9242f](https://github.com/Polymer/core-header-panel/commit/9f9242fc3ba72509028e68ce1153d7c14fe4850a)  do not make it possible to create components, sons of "#mainContainer" in position: fixed
In the previous version the bug was not present.

> Fixes 9f9242f code
> `#mainContainer {transform: translateZ(0);}`
> 
> Consider solution code
> `#mainContainer {transform: translateZ(1);}`
